frontend/src/hooks/useZaakSelection.ts

```diff
@@ -315,6 +302,7 @@ export function useZaakSelection<T = unknown>(
       ? await Promise.all(detailPromises)
       : undefined;
 
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
I have... opinions :D

I think the linter rule points out a code smell here and would suggest using a simple `if { ... } else { ... }` block since you're imperatively doing things/triggering side-effects

Reply:
:+1: I took a conservative approach by disabling the linting rule - you might opt to override it globally if this is a common pattern but then the test error/assertion wouldn't have been revealed.

Reply:
I don't like global linter exlusions so this seems sensible 👍